src/sentry/search/snuba/backend.py

```diff
@@ -427,5 +428,6 @@ def snuba_search(project_id, environment_id, tags, start, end,
             )
 
     return OrderedDict(
-        sorted(((gid, score_fn(data)) for gid, data in group_data.items()), key=lambda t: t[1], reverse=True)
```

Comment:
Not sure why my pre-commit linter triggers this change and the previous committer's didn't